src/wallet/rpcwallet.cpp

```diff
@@ -2309,7 +2309,7 @@ UniValue lockunspent(const JSONRPCRequest& request)
         return true;
     }
 
-    UniValue outputs = request.params[1].get_array();
+    UniValue outputs = !request.params[1].isNull() ? request.params[1].get_array() : NullUniValue;
```

Comment:
Should revert this line too. This is pointless because of the type check above.

I think it's best to stick to just handling the simple cases of missing values being treated differently from null values in this PR and handle the other cases more thoughtfully in another PR. Again, I already have the changes to do this in my multiopt branch, and would be happy to post a followup that builds on your change.